frontend/finance-tracker/src/components/ui/calendar.tsx

```
@@ -1,75 +1,524 @@
+                CaptionLabel: (props) => (
+                    <CaptionLabel
+                        showYearSwitcher={showYearSwitcher}
+                        navView={navView}
+                        setNavView={setNavView}
+                        displayYears={displayYears}
+                        {...props}
+                    />
+                ),
+                MonthGrid: ({ className, children, ...props }) => (
+                    <MonthGrid
+                        children={children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)